qa/L0_lint/test.sh

```diff
@@ -15,12 +15,12 @@ then
   cpplint --root transformer_engine/common/include --recursive transformer_engine/common/include
   echo "Checking C++ files"
   cpplint --recursive --exclude=transformer_engine/common/include transformer_engine
-  cpplint --recursive transformer_engine/pytorch
+  cpplint --recursive transformer_engine/pytorch transformer_engine/tensorflow
```

Comment:
This test is performed in `qa/L0_tensorflow_lint/test.sh`:
```suggestion
  cpplint --recursive transformer_engine/pytorch
```

Reply:
Done.